### ISSUE TITLE
Bug Fixing - Wiki Article Verification

### ISSUE THREAD

**Issue by jaidan18** [RedBlueCarrots/WikiPath#69]
The wiki article verification code recognises redirect and disambiguation pages as articles.

This fix would be nice to have but is not necessary, it just forces users to be precise when entering article names.

**Comment by RadiationOcelot** [comment]
Maybe it's fine if we communicate to the users that they have to be specific with article names?

**Comment by RedBlueCarrots** [comment]
going to put this on hold temporarily - will be better to work on once #18 is done

**Comment by RedBlueCarrots** [comment]
After some discussion with the team, it was decided to allow disambiguation articles - They are separate wikipedia pages after all. This issue will just focus on redirects now
